graspologic/utils/utils.py

```diff
@@ -701,3 +701,79 @@ def plug_in_variance_estimator(x):
         return covariances
 
     return plug_in_variance_estimator
+
+
+def remove_vertices(graph, indices, return_vertices=False):
+    """
+    Remove vertices from an adjacency matrix.
+
+    Parameters
+    ----------
+    graph: np.ndarray, shape (n, n)
+        Adjacency matrix
+    indices: int or list
+        Index/indices of the vertex/vertices to be removed.
+    return_vertex: bool, optional
+        Whether to return the tuple (A, V),
+        where A is the truncated adjacency matrix,
+        V is an array of the removed vertices, by default False
```

Comment:
same comment, "V is an array of the removed vertices" is not quite right. also feels like symbols should be in math or double backtick?

Reply:
also we usually put the default value in the first line of param desc

Reply:
same comment as above, could say "V is an array of the removed adjacency vectors", using [sknetwork's terminology](https://github.com/sknetwork-team/scikit-network/blob/master/sknetwork/embedding/spectral.py#L299)

Reply:
regardless of variable name will probably just want to say something more descriptive here. like "array/matrix/indicince matrix representing the subgraph(s) that were removed by removing vertices blah blah blah"

Reply:
see 2104452, although might still need some improvement.